react-class-components/src/index.jsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class CustomButton extends React.Component {
+
+  render() {
+    return <button>{ this.state }</button>;
```

Comment:
`this.state` is not what we're trying to see in the buttons. it's possible you updated your code after committing or haven't pushed your most recent commit.